hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/CloudBigtableReplicationTask.java

```diff
@@ -220,4 +221,33 @@ static RowMutations buildRowMutations(byte[] rowKey, List<Cell> cellList) throws
     mutationBuilder.buildAndUpdateRowMutations(rowMutationBuffer);
     return rowMutationBuffer;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (!(o instanceof CloudBigtableReplicationTask)) {
+      return false;
+    }
+    CloudBigtableReplicationTask that = (CloudBigtableReplicationTask) o;
+    return Objects.equal(tableName, that.tableName)
+        && Objects.equal(cellsToReplicateByRow, that.cellsToReplicateByRow);
```

Comment:
In general it seems weird to implement equals here

Reply:
My idea was to compare "state" of the object. Connection is not a state and is not owned by this class anyways. Comparing tasks make the tests more compact. 

Otherwise, i will need to use ArgumentCaptor and compare each element of the task in the tests.